hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -928,7 +996,7 @@ public AbfsRestOperation checkAccess(String path, String rwx, TracingContext tra
     abfsUriQueryBuilder.addQuery(QUERY_PARAM_ACTION, CHECK_ACCESS);
     abfsUriQueryBuilder.addQuery(QUERY_FS_ACTION, rwx);
     appendSASTokenToQuery(path, SASTokenProvider.CHECK_ACCESS_OPERATION, abfsUriQueryBuilder);
-    URL url = createRequestUrl(path, abfsUriQueryBuilder.toString());
+    final URL url = createRequestUrl(path, abfsUriQueryBuilder.toString());
```

Comment:
lets check if if blob endpoint API and dfs endpoint API gives same exception -> statusCode, exceptionMessage.

Reply:
2023-04-02 22:49:55,061 DEBUG [JUnit-testFsActionEXECUTE]: services.AbfsIoUtils (AbfsIoUtils.java:dumpHeadersToDebugLog(64)) -   HTTP Response=HTTP/1.1 403 This request is not authorized to perform this operation using this permission. 

The status code for blob endpoint is 403

Reply:
The response code and message is same for both blob and dfs endpoint.